client/components/theme-collection/style.scss

```diff
@@ -19,7 +19,7 @@
 
 	.theme-card__image-container {
 		border-radius: 4px;
-		box-shadow: none;
+		box-shadow: 0 2px 16px 0 rgba(0, 0, 0, 0.15);
```

Comment:
On the Figma, there's also a slightly darker border color: `var( --studio-gray-5 )`.

Reply:
Fixed.